Makefile

```diff
@@ -1,11 +1,5 @@
-ifeq ($(OS), Windows_NT)
-	devnull := nul
-else
-	devnull := /dev/null
-endif
-
 next_version := 1.16.0
-tag := $(shell git describe --exact-match --tags 2>$(devnull))
+tag := $(shell git describe --exact-match --tags 2>git_describe_error.tmp; rm -f git_describe_error.tmp)
```

Comment:
I'm not even sure what this is - it's not something I changed intentionally.  It might have come in when I was pulling (often) from master when the CI system was having issues.  I'll change it back to the way master has it.

Reply:
ah yeah, Looks like this is from another PR. If you rebase master instead of merging you'll avoid these.